### PR TITLE
OSDOCS-3366: Adding steps to check kubelet cert expirations

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OSDOCS-3366

Link to docs preview:
Preview: https://60860--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown.html#graceful-shutdown_graceful-shutdown-cluster

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/graceful-shutdown.adoc
Comment (bergerhoffer) [inline modules/graceful-shutdown.adoc:56]: Can they check a single node, just any node? Or have to check multiple/specifically a control plane node or anything?

**Comment by bergerhoffer** [comment]
@ingvagabund Can you take a first look at this before we send to others to look at? I made some assumptions (that you can log into just any node to check) and adjustments to the suggested commands (to only show the 'notAfter' date instead of also the 'notBefore' date).

I also wasn't confident about what they were actually called vs what the cert that we were already checking for in the procedure was called, if you can check that and let me know.

Preview: https://60860--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown.html#graceful-shutdown_graceful-shutdown-cluster

File: modules/graceful-shutdown.adoc
Comment (ingvagabund) [reply modules/graceful-shutdown.adoc:56]: Given the master nodes can be scaled up/down at any point, we should suggest to actually sync certificates of all nodes by rebooting all master nodes one by one before shutting down the cluster. Effectively making the expiration date almost identical everywhere. Otherwise, all master nodes should be checked and the minimum of all dates picked.

**Comment by ingvagabund** [comment]
> I also wasn't confident about what they were actually called vs what the cert that we were already checking for in the procedure was called, if you can check that and let me know.

Can't tell :( Let's have others to take a look.

File: modules/graceful-shutdown.adoc
Comment (bergerhoffer) [reply modules/graceful-shutdown.adoc:56]: Thanks, I can update to suggest to check all control plane nodes and use the earliest date

Comment (bergerhoffer) [reply modules/graceful-shutdown.adoc:55]: @ingvagabund Updated for this, let me know what you think

**Comment by bergerhoffer** [comment]
@stlaz Can you PTAL at this from the auth side?

@ingvagabund do you know of anyone else we should run this by as well?

**Comment by ingvagabund** [comment]
@rphillips PTAL

**Comment by rphillips** [comment]
LGTM.. the PR says 'on or before' the day of expiration... It is probably safer to say 'before'.

**Comment by bergerhoffer** [comment]
Thanks @rphillips, updated!

**Review by skrthomas — APPROVED** [review]
LGTM! Nice work :)

**Comment by kasturinarra** [comment]
@bergerhoffer i checked the commands and things looked good for me. I would suggest waiting for a review from @sunilcio  and @wangke19 or @gangwgr

**Comment by gangwgr** [comment]
@kasturinarra looks good to me

**Comment by bergerhoffer** [comment]
Thanks @kasturinarra and @gangwgr! 

@sunilcio can you double check as well please?

**Pull request merged**
